panos_upgrade_assurance/exceptions.py

```diff
@@ -0,0 +1,82 @@
+class WrongDataTypeException(Exception):
+    """Used when passed configuration does not meet the data type requirements. Used in all modules."""
+
+    pass
+
+
+class FirewallProxyExceptions(Exception):
```

Comment:
Can we name the base exception classes singular like 
`class FirewallProxyException(Exception):`